doc/charm++/manual.rst

```diff
@@ -2555,173 +2557,196 @@ Examples are in ``examples/charm++/load_balancing`` and
+
+Listed below are load balancers intended for diagnostic purposes:
+
+- **Dummy**: Does nothing, does not move objects at all. (Old: ``+balancer DummyLB``)
+
+- **Random**: Randomly assigns objects to processors. (Old: ``+balancer RandCentLB``)
+
+- **Rotate**: Moves objects to the next available PE every time it is
+  called. It is useful for debugging PUP routines and other migration
+  related bugs. (Old: ``+balancer RotateLB``)
+
+However, not all non-distributed load balancers are currently
+supported by TreeLB, namely the centralized communication-aware load
+balancers:
+
+- **RecBipartLB**: Uses recursive bipartitioning to partition the
+  object communication graph.
+
+- **MetisLB**: Uses `METIS
+  <http://glaros.dtc.umn.edu/gkhome/metis/metis/overview>`__ to
+  partition the object communication graph. METIS is distributed with
+  Charm++, so there is no need to separately get this dependence.
+
+- **ScotchLB**: Uses the `SCOTCH
+  <http://www.labri.fr/perso/pelegrin/scotch/>`__ library for
+  partitioning the object communication graph, while also taking
+  object load imbalance into account. SCOTCH is not distributed with
+  Charm++, so end users must download and build the library from the
+  above link in order to use this load balancer. Because of this
+  dependence, ScotchLB is not built by default; it can be built by
+  running ``make ScotchLB`` in the Charm++ build folder
+  (e.g. ``netlrts-linux-x86_64-smp/``). If SCOTCH is installed in a
+  non-standard location, use the *-incdir* and *-libdir* build time
+  options to point to the include and library directories used,
+  respectively.
+
+In distributed approaches, load data is only exchanged among
+neighboring processors and there is no messaging outside this local
+neighborhood. However, they will not, in general, provide an immediate
+restoration for load balance - the process is iterated until the
+desired load balance can be achieved.
 
 Listed below are the distributed load balancers:
 
--  **NeighborLB**: A neighborhood load balancer in which each processor
-   tries to average out its load only among its neighbors.
-
--  **WSLB**: A load balancer for workstation clusters, which can detect
-   load changes on desktops (and other timeshared processors) and adjust
-   load without interfering with other’s use of the desktop.
-
 -  **DistributedLB**: A load balancer which uses partial information
    about underloaded and overloaded processors in the system to do
    probabilistic transfer of load. This is a refinement based strategy.
 
-An example of a hierarchical strategy can be found in:
-
--  **HybridLB**: This calls GreedyLB at the lower level and RefineLB at
-   the root.
-
-Listed below are load balancers for debugging purposes:
+Custom strategies that are based on CentralLB or HybridBaseLB will
+continue to be supported for now, but support for these will likely be
+dropped in a future release.
```

Comment:
Should add that users should develop their own custom balancers on TreeLB from now on here.

Reply:
Done.